samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -188,9 +188,10 @@ For example, if you are using an nRF9160 DK version 1.0.1, use the following boa
 
 ``nrf9160dk@1.0.1/nrf9160/ns``
 
-This sample also supports placement of the MCUboot secondary partition in external flash for the nRF91x1 DKs, and for nRF9160 DK version 0.14.0 and higher.
-To enable this, add the following parameter to your build command:
+By default,the MCUboot secondary partition is placed in external flash on the nRF91x1 DKs, and on the nRF9160 DK version 0.14.0 and higher.
+To disable this, add the following parameter to your build command:
 
+TODO
```

Comment:
I also completely forgot to fill out this TODO, but what I do here depends on discussion we are having offline